### PR TITLE
dlb, idxd, qat: install LICENSE to initcontainer

### LINKED ISSUE #1604: qat-initcontainer does not have LICENSE installed in root/licenses dir
qat-initcontainer does not have LICENSE installed in root `/licenses/intel-device-plugins-for-kubernetes/LICENSE` dir which results in preflight failure for openshift certification. 

sgx example here -  https://github.com/intel/intel-device-plugins-for-kubernetes/blob/3739fcbd7fa0705c1d873afd20fa10301b1a5d14/build/docker/intel-sgx-initcontainer.Dockerfile#L53
Can we do this for the qat-initcontainer too. Thanks

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
fixes #1604

### CODE REVIEW

**Comment by hj-johannes-lee** [comment]
I added only to qat-initcontainer.
But if it is better to divide the "default_licences.docker" into two different pieces, or change one of the other "libraries", let me know. I did not see any reason to do so.

**Comment by tkatila** [comment]
I did some grepping. Seems like intel-dlb-initcontainer, intel-idxd-config-initcontainer and intel-qat-initcontainer don't copy our LICENSE file. Could you update those as well?

**Comment by hj-johannes-lee** [comment]
> I did some grepping. Seems like intel-dlb-initcontainer, intel-idxd-config-initcontainer and intel-qat-initcontainer don't copy our LICENSE file. Could you update those as well?

Hi, Tuomas! Mentioning `intel-qat-initcontainer` again means that we need to copy something else than the current change?
Isn't "our" LICENSE the file in the root directory of the repo?

**Comment by tkatila** [comment]
> Sure, I can add them to this pr.

:+1: 
 
> But, I am a bit confused about why you mention `intel-qat-initcontainer` again. Do you mean that we need to copy something else than the current change?

AFAIK, current change is fine for qat initcontainer. I was just grepping the project without your change.

> Isn't "our" LICENSE the file in the root directory of the repo?

Yes, but we should have it also in the container images.

File: build/docker/intel-idxd-config-initcontainer.Dockerfile [hunk 1]
```diff
@@ -15,6 +15,8 @@
 ## limitations under the License.
 ###
 FROM debian:unstable-slim
+COPY ./LICENSE .
+RUN install -D /LICENSE /licenses/intel-device-plugins-for-kubernetes/LICENSE && rm /LICENSE
```
Comment (mythi) [inline build/docker/intel-idxd-config-initcontainer.Dockerfile:19]: `COPY` is enough: "If <dest> doesn't exist, it's created, along with all missing directories in its path."

Comment (hj-johannes-lee) [reply build/docker/intel-idxd-config-initcontainer.Dockerfile:19]: OK ;) Thanks!

File: build/docker/intel-qat-initcontainer.Dockerfile [hunk 2]
```diff
@@ -39,6 +39,7 @@ FROM ${GOLANG_BASE} as builder
 ARG DIR=/intel-device-plugins-for-kubernetes
 WORKDIR $DIR
 COPY . .
+RUN install -D ${DIR}/LICENSE /install_root/licenses/intel-device-plugins-for-kubernetes/LICENSE
```
Comment (mythi) [inline build/docker/intel-qat-initcontainer.Dockerfile:42]: I noticed that this line is not in a good place. What if someone overrides `ROOT` build-arg to be something else than `/install_root`

Comment (mythi) [reply build/docker/intel-qat-initcontainer.Dockerfile:42]: let's go with this.

**Review by mythi — APPROVED** [review]

**Pull request merged**